components/x-teaser-timeline/src/lib/transform.js

```diff
@@ -165,15 +165,19 @@ export const buildModel = ({
 		latestItemsTime,
 		latestItemsAgeHours
 	})
-
 	if (itemGroups.length > 0 && customSlotContent) {
-		const insertPosition = Math.min(customSlotPosition, items.length)
-		const insert = getGroupAndIndex(itemGroups, insertPosition)
-		const copyOfItems = [...itemGroups[insert.group].items]
+		customSlotContent = Array.isArray(customSlotContent) ? customSlotContent : [customSlotContent]
```

Comment:
Maybe the variable names customSlotContent and customSlotPosition can be pluralised now that they are respectively an array of slots, and an array of position.
I would also consider improving the prototype of the `X-Teaser-Timeline` component to take a `customSlots` prop like
```
customSlots: Record<position, content>
```
which would still allow us to loop over the keys (slot positions) or over the values (slot contents).

Reply:
I agree with the above, though this would be better for a future refactor. I don't want to make this a breaking change and doing so will but in order to make this change a non-breaking change, I need to think about this some more.